src/App.js

```diff
@@ -1,25 +1,61 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import "@baltimorecounty/dotgov-components/lib/styles/dotgov.min.css";
+import "./App.css";
+
+import { Config } from "@baltimorecounty/javascript-utilities";
+import { FilterList } from "@baltimorecounty/react-filter-list";
+import PoliceNewsRoomCard from "./components/PoliceNewsRoomCard";
+import React from "react";
+
+const { setConfig, getValue } = Config;
```

Comment:
not blocking, but something ot think about, i'd like to have a better way of doing this config part, maybe just dump it in it's own class, or build another method in the JavaScript utils project, so we don't have this 20 liner in every app.js